### PR TITLE
Acrs 47 full name

### PR DESCRIPTION
**Pull request by michaelhartddat**
What?

[ACRS-47 - What is your full name](https://collaboration.homeoffice.gov.uk/jira/browse/ACRS-47)
Adds the `/full-name` page to the `acrs/` steps

Why?

The `/full-name` page is a requirement of the flow.

### CODE REVIEW

File: apps/acrs/translations/src/en/fields.json [hunk 1]
```diff
@@ -20,15 +20,8 @@
...
-      "label": "SKELETON: full-name",
-      "options": {
-        "under-18": {
-          "label": "SKELETON: Referrer DOB < 28-09-2023 (under 18)"
-        },
-        "over-18": {
-          "label": "SKELETON: Referrer DOB >= 28-09-2023 (over 18)"
-        }
-      }
+    "label": " ",
```
Comment (jamiecarterHO) [inline apps/acrs/translations/src/en/fields.json:23]: You will need to use the text of the question as the label for accessibility purposes here. It should then show up as the H1 if you add `isPageHeading: true` to the relevant block in acrs/fields/index.js.

File: apps/acrs/translations/src/en/pages.json [hunk 2]
```diff
@@ -64,7 +64,7 @@
     "header": "SKELETON: /complete-as-referrer"
   },
   "full-name": {
-    "header": "SKELETON: /full-name"
+    "header": "What is your full name?"
```
Comment (jamiecarterHO) [inline apps/acrs/translations/src/en/pages.json:67]: If you use `isPageHeading` you can remove the `full-name` block from the main part of pages.json.

**Review by jamiecarterHO — CHANGES_REQUESTED** [review]
Very nice. Lovely annotation on the over18 functions.
Just one suggestion around flipping the way the question/heading is added to the full-name question page

File: apps/acrs/translations/src/en/fields.json (hunk 1 above)
Comment (michaelhartddat) [reply apps/acrs/translations/src/en/fields.json]: Good to know 👍

**Review by jamiecarterHO — APPROVED** [review]
Looks great

**Pull request merged**